src/commands/ssh.ts

```diff
@@ -43,9 +43,9 @@ export const sshCommand = (yargs: yargs.Argv) =>
           describe: "Reason access is needed",
           type: "string",
         })
-        .option("account", {
+        .option("parent", {
           type: "string",
-          describe: "The account on which the instance is located",
+          describe: "The parent resource on which the instance is located",
```

Comment:
Nit: I wonder if this should be a little more descriptive, maybe by way of brief examples? Maybe something along the lines of, "e.g., account ID for AWS, resource group ID for Azure"?

Reply:
updated 👍